saleor/order/calculations.py

```diff
@@ -514,3 +523,17 @@ def order_undiscounted_total(
     currency = order.currency
     order, _ = fetch_order_prices_if_expired(order, manager, lines, force_update)
     return quantize_price(order.undiscounted_total, currency)
+
+
+def dd(line, line_total, amount: TaxedMoney, tax_rate: int, prices_entered_with_tax):
```

Comment:
Is this function expected?

Reply:
it was a helper only for development but it's not needed, removed.